app/modules/intelligence/tools/web_tools/code_provider_tool.py

```
@@ -0,0 +1,250 @@
+
+            example:
+            {
+                "repo_name": 'owner/repo',
+                "issue_number": 123,
+                "is_pull_request": true
+            }
+
+        Returns dictionary containing the issue/PR content, metadata, and success status.
+        """
+
+    gh_token_list: List[str] = []
```

Ruff (0.14.4)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)